runit-java8-bionic/collectd-jmx.conf

```diff
@@ -121,7 +121,7 @@ LoadPlugin java
     </MBean>
 
     <Connection>
-      Host "<HOSTNAME>"
+      Host <HOSTNAME>
```

Comment:
The sed line in set_jmx_hostname does not replace "<HOSTNAME>".
I tested this by executing bash in a running container. I removed the quotes and executed "set_jmx_hostname". It performed the replace as expected.